lessons/pydata/databases/index.ipynb

```diff
@@ -0,0 +1,1078 @@
+{
```

Comment:
<p>To porovnání operací v pandas  vs SQL je supr nápad.</p><p> </p><p>Možná by se někdo mohl ptát proč dva způsoby? Na to by se hodilo nějaký porovnání s tím kdy použít co. Třeba jako když jsi chci jenom něco rychlého ověřit, tak to udělám v SQL. A tak podobně.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>Já bych zde jako příklad uvedl možnost, kdy je databáze obrovská a leží někde na internetu a já tím, že data zpracuji ještě pomocí SQL nemusím tahat takové množství dat k sobě. Ušetřím tím čas i následné zpracování v pandasu. Nehledě na to, že těch dat může být tolik, že si s nimi lokálně ani neporadím.</p>